doc/synchronization.md

```diff
@@ -0,0 +1,356 @@
+> Quotations are used for notes.
+
+> This document is work-in-progress.
+> Intentions of this effort and document are: to sum up the behavior
```

Comment:
Could be 'to summarise' rather than 'to sum up'. Sum up sounds more like a conclusion.